### PR TITLE
159-curation-guidelines

### PR DESCRIPTION
**Pull request by jahilton**
Addresses #159

### CODE REVIEW

**Comment by BAevermann** [comment]
I think it reads well. Are we going to make those collection lists publicly viewable?

**Review by BAevermann — APPROVED** [review]

**Comment by jahilton** [comment]
>Are we going to make those collection lists publicly viewable?

@BAevermann the sheet is currently publicly viewable. And I don't see anything like private collection links shared that shouldn't be. But it's also not organized well or particularly useful that we should be actively sharing the link broadly.

File: guidelines/curation_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (brianraymor) [inline guidelines/curation_guidelines.md:6]: RE _currently being curated_ - I would not expect the Datasheet sheet to expose private collections so the comment is confusing. I could imagine that the sheet might include the DOI and curator name for in progress items.

Comment (brianraymor) [inline guidelines/curation_guidelines.md:6]: RE _currently being curated_ - I would not expect the Datasheet sheet to expose private collections so the comment is confusing. I could imagine that the sheet might include the DOI and curator name for in progress items.

Comment (jahilton) [reply guidelines/curation_guidelines.md:6]: The Dataset sheet only includes DOI and curator team for in progress items, so this expectation is accurate.
Does this wording clarify that?
>The curator should then check the Dataset sheet for the **DOI of the** primary study and...

Comment (brianraymor) [reply guidelines/curation_guidelines.md:6]: Yes.

Comment (jahilton) [reply guidelines/curation_guidelines.md:6]: Addressed in 7fc87fd96ddb9d7eddbe34dd1b617d4139549ac6

**Review by brianraymor — APPROVED** [review]

**Pull request merged**